Function/model/BookTableModel.cs

```diff
@@ -0,0 +1,24 @@
+using Azure;
+using Azure.Data.Tables;
+
+namespace AzureFunctionExample.Model;
+public class BookTableModel : ITableEntity
```

Comment:
This model is inheriting from `ITableEntity`, which makes it possible to write objects instantiated from this model directly to the table via the table client. This also makes it possible to read a row from the table client to this model.